documentation/release_notes.rst

```diff
@@ -8,6 +8,92 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.7.0
+===============
+News
+------------
+
+New Features
+------------
+- Improved performance of the ``reduce``, ``min_element``, ``max_element``, ``minmax_element``, ``is_partitioned``, and ``lexicographical_compare`` algorithms with device policies. 
```

Comment:
May be make sense to combine all algorithms together under text "Improved performance" and sort by name?

Reply:
I think so.  The questions are whether @dmitriy-sobolev's qualification of performance improvement of sort performance should be kept separately, and if the end of the sentence should be "algorithms with device policies" or "algorithms on GPU devices."

Reply:
I've made this change, leaving the merge_sort item separate.

Reply:
The sort optimizations targeted GPU devices, and the overall effect on CPU devices is negligible (some data sizes are now sorted faster and some slower).